lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -0,0 +1,93 @@
+"""
+Tests for Blocks Views
+"""
+
+from django.core.urlresolvers import reverse
+from lettuce import world
+
+from student.models import CourseEnrollment
+from student.tests.factories import CourseEnrollmentFactory
+from xmodule.modulestore.tests.django_utils import SharedModuleStoreTestCase
+from xmodule.modulestore.tests.factories import ToyCourseFactory
+
+
+
+class CourseApiTestViewMixin(object):
+    """
+    Mixin class for test helpers for Course API views
+    """
+    @classmethod
+    def create_course(cls):
+        """
+        Create a sample course
+        """
+        return ToyCourseFactory.create()
+
+    def create_user(self, username, email, password, is_staff):
+        return world.UserFactory(
```

Comment:
Same comment as above regarding using `UserFactory` from the `student.tests` module.